cpp/src/Slice/Parser.cpp

```diff
@@ -1050,12 +1052,6 @@ Slice::Contained::Contained(const ContainerPtr& container, const string& name)
       _container(container),
       _name(name)
 {
-    ContainedPtr cont = dynamic_pointer_cast<Contained>(_container);
-    if (cont)
-    {
-        _scoped = cont->scoped();
-    }
-    _scoped += "::" + _name;
```

Comment:
There isn't much point to `_scoped` now.

Most of the calls to `scoped()` and `scope()` are for mapped identifiers, and now we have to compute their values on the fly. So, there's even less point to caching the `_scoped` name up-front. So, I removed it.